tests/integration/base_object_test.go

```diff
@@ -51,20 +51,14 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	require.Nil(s.T(), cobsErr)
 	logger.Infof("Object store created successfully")
 
-	/* TODO: Reenable this test after we have the object user CRD
-
 	logger.Infof("Step 1 : Create Object Store User")
-	initialUsers, _ := helper.ObjectClient.ObjectListUser(storeName)
-	_, cosuErr := helper.ObjectClient.CreateUser(storeName, userid, userdisplayname)
+	cosuErr := helper.ObjectUserClient.Create(namespace, userid, userdisplayname, storeName)
 	require.Nil(s.T(), cosuErr)
-	usersAfterCreate, _ := helper.ObjectClient.ObjectListUser(storeName)
-	require.Equal(s.T(), len(initialUsers)+1, len(usersAfterCreate), "Make sure user list count is increased by 1")
-	getuserData, guErr := helper.ObjectClient.ObjectGetUser(storeName, userid)
-	require.Nil(s.T(), guErr)
-	require.Equal(s.T(), userid, getuserData.UserID, "Check user id returned")
-	require.Equal(s.T(), userdisplayname, *getuserData.DisplayName, "Check user name returned")
+	require.True(s.T(), helper.ObjectUserClient.UserSecretExists(namespace, userid, storeName), "make sure user secret was created")
```

Comment:
in addition to checking for existence what if you also return the user attributes and check that they match what was originally generated?

Reply:
That was what I was originally trying to do, but was having a lot of trouble with the rgw client. The json unmarshal at: https://github.com/rook/rook/blob/bf2759e317c44c0ad0aaf635e04cbd72a002a5a0/pkg/daemon/ceph/rgw/user.go#L49 

was somehow getting an Input starting with `Unable to use a TTY - input is not a terminal or the right kind of file` before the user list JSON that I couldn't figure out. Perhaps it was how I was building the rgwcontext? Have you run into this before?

Reply:
The `result` included that string message instead of json? Or was that included before the json and messed up the json deserialization? You could try running the same command in the rook toolbox and see if it returns that string. The log file should show you the exact command that was run.

Reply:
The `result` had that string message, a newline, and then the JSON. So it messed up the json deserialization. The error was:
`Cannot unmarshal json response: invalid character 'U' looking for beginning of value` referring to the "U" at the beginning of `Unable to use a TTY...`

Reply:
i wonder if #2247 fixed this issue, which was merged just a couple days ago.

Reply:
I may have given up on that handling before rebasing against that commit. I'll try again with that commit included and report back.

Reply:
It appears as though #2247 has fixed this issue. I was able to write and pass tests for both userID and displayname. New commit inbound shortly...